src/core/createInstance.js

```diff
@@ -24,13 +24,15 @@ export default (
 ) => {
   let componentRegistry;
   let configurationFailed = false;
+  let suppressErrors;
 
   const logCommand = ({ enabled }) => {
     // eslint-disable-next-line no-param-reassign
     logController.logEnabled = enabled;
   };
 
   const configureCommand = options => {
+    ({ suppressErrors } = options);
```

Comment:
I would really like to add a config validator for this option, along with the `log` option, but can't, because of when the config validation is run. I've logged an issue here: https://jira.corp.adobe.com/browse/CORE-32996

Reply:
Also, we need to add tests around the core modules generally, which I've also logged here: https://jira.corp.adobe.com/browse/CORE-32995

Reply:
Thank you for logging the tickets @Aaronius !